src/pages/Admin/Details/Form/DetailsForm.tsx

```diff
@@ -295,16 +295,16 @@ const DetailsForm: React.FC<DetailsFormProps> = ({ venue, worldId }) => {
       Object.fromEntries(
         filteredWorlds
           .filter(({ id }) => !(venueId === id))
-          .map((world) => [world.id, world])
+          .map((world) => [world.slug, world])
```

Comment:
Filtering should still be done by id than by slug, but at this point, `DetailsForm` is no longer used for creating spaces 🎉 so it's kind of a moot point